sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

```diff
@@ -1,34 +1,194 @@
 # Azure KeyVault Administration client library for .NET
 
-Content forthcoming
+Azure Key Vault is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, or certificates. A vault is a logical group of secrets.
+
+The Azure Key Vault administration library clients supports administrative tasks such as full backup / restore and key level role based access control (RBAC).
 
 ## Getting started
 
-Content forthcoming
+### Install the package
+Install the Azure Key Vault administration client library for .NET with [NuGet][nuget]:
 
-### Prerequisites
+```PowerShell
+Install-Package Azure.Security.KeyVault.Administration
```

Comment:
This needs to have the `--version <actual preview version we need to keep up-to-date>` while in preview. We only take it off once we GA and for servicing releases (i.e. existing libraries will have that for previews going forward).